pointsbot.js

```diff
@@ -31,12 +32,16 @@ function handleDish(event, room, client, auth) {
   if (event.getSender() == `@${process.env.BOT_USER}:matrix.org`) { // we sent the message.
     return;
   }
+  if (event.getContent().formatted_body) {
+    message = event.getContent().formatted_body;
+  }
   if (message.trim()[0] == ">") { //quoting another user, skip the quoted part
     matched = true;
     message = message.split("\n\n")[1];
   }
 
-  const re = /!\s*dish\s+(\S+)\s+(\S+)\s+points\s+to\s+(\S+)\s+for\s+([^\n]+)/gi;
+
+  const re = /!\s*dish\s+(\S+)\s+(\S+)\s+points\s+to\s+([^\n]+)\s+for\s+([^\n]+)/gi;
```

Comment:
I actually think that this already handles the condition on line 12, right? https://github.com/Giveth/giveth-bot/pull/24/files#diff-d656cec8a77f5db4f51c4a44749dc151R12

Reply:
The `\s` right after `!`

Reply:
Sorry, I'm not the most proficient in regex, so I can't explain why, but no it doesn't match the string which is why I added the other condition.

Reply:
Oh, nevermind. Just read through it again. The condition is there to match all commands even with a space between "!" and the command. It's done before even executing the regex, and to match commands.

Reply:
The "changed" message without the space isn't sent to the regex anyway, so it successfully matches on the unchanged message. But to register the command we still need to remove the space.

Reply:
I agree with @FrederikBolding seems alright to me. I'll merged this